src/test/java/com/axlabs/neo/grantshares/GovernanceMembersTest.java

```diff
@@ -90,9 +72,8 @@ public static void setUp() throws Throwable {
     //region ADD MEMBER
     @Test
     public void fail_calling_add_member_directly() throws Throwable {
-        Hash256 tx = gov.invokeFunction(ADD_MEMBER, hash160(bob)).signers(AccountSigner.calledByEntry(alice))
-                .sign().send().getSendRawTransaction().getHash();
-        assertAborted(tx, "Method only callable by the contract itself", neow3j);
+        Exception e = assertThrows(TransactionConfigurationException.class, () -> gov.invokeFunction(ADD_MEMBER, hash160(bob)).signers(AccountSigner.calledByEntry(alice)).sign());
```

Comment:
Suggestion: Use the actual exception type `TransactionConfigurationException` instead of the parent class.